src/send/SendConfirmation.tsx

```diff
@@ -97,6 +101,8 @@ function SendConfirmation(props: Props) {
     },
   } = props.route.params
```

Comment:
Seems like this screen still accepts tokenAddress as a param. Should this be updated to accept tokenId instead?
Also it accepts an `inputAmount` (can be either token or local currency amount based on the `amountIsInLocalCurrency` param) and `tokenAmount` but in the new flow the user is only allowed to enter token amounts, so some of these params seem redundant in the new flow. 
I think we should just make a new screen for the new send flow with just the required params and tokenId. There maybe some duplication but it will avoid a bunch of boolean based rendering and make cleanup easier when the new flow is rolled out to 100%.

Reply:
I think making a new screen might be difficult to coordinate, @jophish has changes made to `SendConfirmation` [here](https://github.com/valora-inc/wallet/pull/4306/files#diff-b79fcaffc5d8f13b96f758016a7d2d6ee2a3f180d4b785b0c112793f6cfc7cf2) in #4306 which adds in a tokenId param as well as other changes for multichain, so I'm not sure that it would be easy to create a new screen and make sure it has my changes as well as the necessary multichain changes from Joe.

Reply:
From a quick look at the changes on Joe's PR, it seems like there'd be a messy merge conflict with the changes you have currently. You could just create a new screen with the new UI that just directly supports the multichain params, and we also get the benefit of dropping unnecessary params for the new screen and makes it easier to cleanup once the new stuff is rolled out. But I'll leave it up to you since you're closer to the details

Reply:
Yeah this is kind of tricky. I think it depends on how onerous the merge conflict is? I think I'd probably prefer keeping the screens as one, since if we separate them out, we'd have to duplicate the multichain changes from my PR in your new screen, like you said. Alternatively, if there's a way to refactor stuff so that we minimize duplication, that could be nice, but that may not be possible/end up being even more clutter than having two separate screens.